packages/components/src/Notification/InlineNotification/_docs/InlineNotification.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import { Meta, StoryObj } from "@storybook/react"
+import {
+  InlineNotification,
+  InlineNotificationProps,
+} from "../InlineNotification"
+
+const DEFAULT_PROPS: Partial<InlineNotificationProps> = {
+  headingProps: {
+    children: "Informative",
+    tag: "span",
+    variant: "heading-6",
+  },
+  type: "informative",
+  children:
+    "Lorem ipsum dolor sit amet consectetur adipisicing elit. Perspiciatis quibusdam natus doloremque",
+}
+
+const meta = {
+  title: "KAIO-staging/Notifications/Inline Notification",
+  component: InlineNotification,
+  argTypes: {
+    isSubtle: { type: "boolean" },
+    forceMultiline: { type: "boolean" },
+    noBottomMargin: { type: "boolean" },
```

Comment:
You don't need these anymore, Cass' change should've fixed this

Reply:
Oh nice, thanks for the tip 👍 have removed and it works as expected